docs/install-and-run.md

```diff
@@ -285,3 +285,40 @@ compatible with EarlGrey.
 can run it by just clicking on the **run** icon or by Ctrl+clicking the test name and then selecting
 **Test "testPresenceOfKeyWindow"**. Because this is a regular unit test, you can place breakpoints in
 your test and in your application code and use the built-in tools seamlessly.
+
+### Ruby Installation
+
+If you already have a working Ruby install, skip this part.
```

Comment:
Could you please number each step and then indent 3 spaces for every step?

For eg.

### Ruby Installation

If you already have a working Ruby install, skip this part.

   1. Install Ruby Env.
      ```
          brew update
          brew install rbenv
      ```